napps/metadata.md

```diff
@@ -55,6 +58,18 @@ Metadata
 | priority            | topology    | link       | float | Link's priority, used for traffic engineering | 02.02.2022 | pathfinder                    |
 | delay               | topology    | link       | float | Link's latency in ms                          | 02.02.2022 | pathfinder                    |
 
+
+Internal Metadata
+-----------------
+
+
+| Metadata              | Napp        | Object     | Type  | Description                                   | Date       | Used by                       |
+|-----------------------|-------------|------------|-------|-----------------------------------------------|------------|-------------------------------|
+| liveness_status       | topology    | link       | str   | Link's liveness status  up\|down              | 06.30.22   | topology, core                |
+| last_status_change    | topology    | link       | float | Link's last status change timestamp           | 06.30.22   | topology                      |
+| last_status_is_active | topology    | link       | bool  | Whether Link's last status is active or not   | 06.30.22   | topology                      |
```

Comment:
I wonder if it wouldn't be simple if instead of `last_status_is_active` we just store `last_status` (which could also store other values in the future (in maintenance, maybe?) - not sure if it is actually useful).

Reply:
@italovalcy great idea, I've mapped this enhancement for the next release https://github.com/kytos-ng/topology/issues/99. Also, there's another similar opportunity with `last_status_change`, currently [it stores a float epoch](https://github.com/kytos-ng/topology/blob/master/controllers/__init__.py#L290), which works great. But, if it were stored as datetime, especially since it's a metadata value, it would provide a better UX for providers who could be able to read it without conversions, and also on `MongoDB` its type would be `ISODate` which would enable queries based on datetime constraints without conversions, I haven't mapped this as an enhancement though, other than documenting on that comment that I linked here. Let me know what you think and if we should map this enhancement too.

Reply:
Appreciated your review and inputs.

Reply:
I've mapped it here https://github.com/kytos-ng/topology/issues/104

Reply:
@italovalcy thanks for reviewing and the suggestions, once these new two issues are addressed we can update this page again.